src/image_selector.py

```diff
@@ -103,10 +104,49 @@ def __init__(self):
         # set the first image by default
         self.change_image(self._images[0], 0)
 
+        # Shortcut for toggling Deviant checkbox
+        self.toggle_deviant_shortcut = QShortcut(QKeySequence("Ctrl+D"), self)
```

Comment:
how feasible is it to make this in its own function? this setup function is huge already

```
def setup_shortcuts():
 # stuff
 ```

Reply:
Sure! Updated.